apps/tup-cms/src/taccsite_cms/settings_custom.py

```diff
@@ -205,6 +205,12 @@
 # https://github.com/django-cms/django-filer/blob/2.0.2/docs/permissions.rst
 FILER_ENABLE_PERMISSIONS = True
 
+# https://github.com/django-cms/djangocms-text-ckeditor
+CKEDITOR_SETTINGS = {
+    'autoParagraph': True, # Core-CMS had set this to False
+    'stylesSet': 'default:/static/js/addons/ckeditor.wysiwyg.js',
+    'contentsCss': ['/static/djangocms_text_ckeditor/ckeditor/contents.css'],
+}
```

Comment:
**Explanation.** I must duplicate [these two settings from Core-CMS](https://github.com/TACC/Core-CMS/blob/v4.5.0/taccsite_cms/settings.py#L631-L632), because settings are not smart enough to merge lists.